docs/src/content/docs/fr/guides/overriding-components.mdx

```diff
@@ -24,6 +24,10 @@ Redéfinir les composants par défaut de Starlight peut être utile lorsque :
 1. Choisissez le composant de Starlight que vous souhaitez redéfinir.
    Vous pouvez trouver une liste complète des composants dans la [référence des redéfinitions](/fr/reference/overrides/).
 
+	 :::tip
+	 Vous ne savez pas quel composant vous devez remplacer ? Utilisez la [carte interactive des remplacements de Starlight](https://starlight-overrides-map.netlify.app/) pour découvrir les noms des composants de l'interface utilisateur de Starlight.
```

Comment:
```suggestion
	 Vous ne savez pas quel composant vous devez redéfinir ? Utilisez la [carte interactive des redéfinitions de Starlight](https://starlight-overrides-map.netlify.app/) pour découvrir les noms des composants de l'interface utilisateur de Starlight.
```

Considering "override" in the English text is a 1:1 match with the page title and the feature described, we should do the same in the French version and use "redéfinition" like in the current page or the entire Starlight docs.